libs/auth-api-lib/src/lib/personal-representative/entities/dto/personal-representative.dto.ts

```diff
@@ -6,69 +6,89 @@ import {
   IsArray,
   ArrayNotEmpty,
 } from 'class-validator'
-import { ApiProperty } from '@nestjs/swagger'
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger'
 import { Type } from 'class-transformer'
 import { PersonalRepresentativeRightTypeDTO } from './personal-representative-right-type.dto'
+import { PersonalRepresentative } from '../models/personal-representative.model'
+import { PersonalRepresentativePublicDTO } from './personal-representative-public.dto'
 
 export class PersonalRepresentativeDTO {
   @IsString()
   @IsOptional()
-  @ApiProperty({
+  @ApiPropertyOptional({
     example: 'guid',
   })
   readonly id?: string
 
   @IsString()
-  @IsOptional()
   @ApiProperty({
     example: 'personal_representative_for_disabled_person',
   })
-  readonly personalRepresentativeTypeCode?: string
+  readonly personalRepresentativeTypeCode!: string
 
   @IsString()
   @IsNotEmpty()
   @ApiProperty({
-    example:
+    example: '99',
+    description:
       'contractId from personal representative contract system from personal representative contract system or other external system used to create personal rep contracts',
   })
   readonly contractId!: string
 
   @IsString()
   @IsNotEmpty()
   @ApiProperty({
-    example:
+    example: 'usernameA',
+    description:
       'userId(AD) from personal representative contract system or other external system used to create personal rep contracts',
   })
   readonly externalUserId!: string
 
   @IsString()
   @IsNotEmpty()
   @ApiProperty({
-    example: 'nationalId of Personal Representative',
+    example: '0123456789',
+    description: 'nationalId of Personal Representative',
+    pattern: '^(0-9){10}$',
```

Comment:
In other places you are using `d` instead of `(0-9)` so update to use the same regex.